crates/re_viewer/src/ui/selection_panel.rs

```diff
@@ -46,7 +49,7 @@ pub(crate) struct SelectionPanel {
 /// timeline, then use the blueprint. Otherwise use the recording.
 // TODO(jleibs): Ideally this wouldn't be necessary and we could make the assessment
 // directly from the entity_path.
-fn guess_query_and_store_for_selected_entity<'a>(
+pub(crate) fn guess_query_and_store_for_selected_entity<'a>(
```

Comment:
I suppose this partially comes back to "can override paths exist in the data store" if the answer is yes this becomes both necessary and fragile. If no, we can check if an path is an override and pick the right store making this not a "guess" but a "know"

Reply:
This is actually sort of the opposite -- not the location of the override, but the location of the overridden data.  However, there's no meaningful way (yet) to override data that lives in the blueprint store.

This is actually an unnecessary copy-paste from some other selection panel logic. Removed the guess part and added a comment.